src/components/dace_interface.ts

```diff
@@ -29,6 +29,9 @@ enum InteractionMode {
     APPLY,
 }
 
+const MIN_SAFE_VERSION = '0.16.0';
```

Comment:
Same question as before. How does typescript do version checks in general?

Reply:
The semver package appears to be the production-grade industry standard - I adopted that too now